dao.md

```diff
@@ -0,0 +1,40 @@
+
+[OceanDAO](https://www.oceanprotocol.com/dao) is a key component that brings decentralization and scale to participants that are driving traction into the Ocean ecosystem. We’re growing the OceanDAO team. If you love to work with crypto, build natively on web3, and to engage deeply with an enthusiastic community, we want to hear from you.
+
+
+## Responsibilities
+* You will design & build for OceanDAO core software, from the level of blockchain to the level of dapps. Integrating governance, treasury management, viral mechanisms, and whatever else is needed for OceanDAO to be concretized.
+* The scope of the software is well defined but may evolve over time as DAOs are nascent, e.g. integrating decentralized courts, and decentralizing funding mechanisms. You will co-plan and co-design the product with the OceanDAO team in an agile, open-source development process.
+
+
+## Minimum Qualifications
+* 4+ years of software experience
+* 3+ years backend experience, including NodeJS
+* 6+ months of Solidity experience
+* Experience working in technical/development roles in the crypto projects
+* Process-driven, proactive - able to report on your activity and communicate well with the team and DAO (community).
+
+
+## Nice-to-haves
+* 2+ years working with cloud infrastructure, supporting devops.
+* 1+ years CI/CD experience (e.g. Travis, tox)
+* You’re obsessed with blockchain and DAOs. You play with dapps, vote on proposals, or have contributed code to a DAO.
+* Excellent writing and communication skills. You will engage with the community, receive feedback, identify usage patterns, and communicate macro outcomes.
+* Successfully delivered on promises, comfortable working autonomously and within a team
+* Differentiate good and bad feedback to find real pain points to improve product/program based on learnings and manage and update scripts and reward function accordingly
+* Proven experience in building meaningful technical communities around projects and ability to build and manage relationships with key stakeholders
+* Stellar research skills
+* Preferred work hours: 9-5 on North American time zones. We start early to sync with the EU team (7am PDT).
+* We don’t expect you to be an expert in all of these. Rather, we’d love to see that you’re interested in them, perhaps know one or two as a starting point, and most importantly, that you’ll learn more about them as you grow with us professionally and personally.
+
+
+
+## Your Next Step?
+If you love to build things and care deeply about the future of data, we want to hear from you! Please email [devjobs@oceanprotocol.com](mailto:devjobs@oceanprotocol.com), with:
+* your cover letter - why you’d like to join
+* your CV
```

Comment:
Also ask for eth address(es) or ens names. Call it optional. This will be super useful for us though!